src/libraries/System.Private.DataContractSerialization/src/System/Xml/XmlUTF8TextWriter.cs

```diff
@@ -68,11 +68,8 @@ internal class XmlUTF8NodeWriter : XmlStreamNodeWriter
             (byte)'e', (byte)'n', (byte)'c', (byte)'o', (byte)'d', (byte)'i', (byte)'n', (byte)'g', (byte)'=', (byte)'"', (byte)'u', (byte)'t', (byte)'f', (byte)'-', (byte)'8', (byte)'"',
```

Comment:
Would changing the 3 arrays above to `""u8` literals with `.ToArray()` be a good idea? The codegen's slightly worse but it shouldn't matter in initialization.

Reply:
Sure